packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -370,6 +363,8 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
             to: transfer.to,
             type: transfer.type,
             value: transfer.totalValue,
+            id: transfer.id,
+            token: transfer.token,
```

Comment:
pass back token data that we have from blockbook to make nft assets as they don't exist in the asset service